vm/devices/storage/scsidisk/src/lib.rs

```diff
@@ -947,6 +947,7 @@ impl SimpleScsiDisk {
                     | ScsiError::UnsupportedVpdPageCode(_)
                     | ScsiError::SrbError
                     | ScsiError::Disk(DiskError::InvalidInput)
+                    | ScsiError::Disk(DiskError::Hv(_))
```

Comment:
This error could rectify on retry, right? I don't think we want to return this as a INVALID REQUEST / INVALID CDB. We can use something like insufficient resources.

I think we want to return SCSI CHECK_CONDITION, Sense Key 0x2 NOT READY, Add'l Sense Key 0x04 (LOGICAL UNIT NOT READY, CAUSE NOT REPORTABLE)

Reply:
You mean the pin/unpin may succeed if retry? I just follow the same logic when we fail on allocate double buffer today, we return InvalidInput (in nvme_driver the error is TooLarge).

Reply:
Yeah, I think it would.